LightBlue.MultiHost/EditRoleView.xaml

```diff
@@ -62,8 +62,8 @@
                           SelectedValuePath="Value"/>
             </Grid>
             <StackPanel Orientation="Horizontal" HorizontalAlignment="Right">
-                <Button x:Name="BtnOk" Margin="3" Content="Save" Width="100" Height="30" Style="{StaticResource AccentedSquareButtonStyle}" Click="OkButton_Click" HorizontalAlignment="Right"/>
-                <Button x:Name="BtnCancel" Margin="3" Content="Cancel" Width="100" Height="30" Style="{StaticResource AccentedSquareButtonStyle}" Click="CancelButton_Click" HorizontalAlignment="Right" />
+                <Button x:Name="BtnOk" Margin="3" Content="Save" Width="100" Height="30" Click="OkButton_Click" HorizontalAlignment="Right"/>
```

Comment:
This was missed in the mahapps upgrade, the correct mapping is `MahApps.Styles.Button.Square.Accent`
ref: https://mahapps.com/docs/guides/migration-to-v2.0